### PR TITLE
Windows: remove obsolete winmath.h

### PR DESCRIPTION
**Pull request by sethrj**
This TriBITS-distributed file seems to have been necessary at some point in the distant past for successful Windows builds, but it now causes build errors:
```
    D:\nfr\scale\TriBITS\tribits\win_interface\include\winmath.h(155): error: function "tgammal" has alread

      inline long double tgammal(long double z){
```

Without the winmath includes, Intrepid successfully builds on VS2015 (with an older Trilinos version) and VS2017.

@bartlettroscoe 

## Motivation

Building on Windows with newer versions of Visual Studio.

### CODE REVIEW

**Comment by bartlettroscoe** [comment]
@sethrj, looks fine to me but I think that at least one @trilinos/intrepid and @trilinos/intrepid2 developer should approve this before it gets merged.

But I will go ahead and approve this so that the PR builds can fire off.

**Review by bartlettroscoe — APPROVED** [review]
Approving so that the PR builds will fire off so this can be tested.  Otherwise, some @trilinos/intrepid and @trilinos/intrepid2 developer should approve before this gets merged.

**Comment by sethrj** [comment]
Thanks @bartlettroscoe ! It looks like the original `winmath.h` includes are actually attributed to @billhoffman in a4dbd013f25ea4a80a6d782e26c20e8a61db8889 11 years ago.

**Review by bartlettroscoe — APPROVED** [review]
Given that this is a very simple change that only impacts Windows builds (for which there are no automated builds of Trilinos right now posting to CDash) and given that no  @trilinos/intrepid  or @trilinos/intrepi2 developer has objected to this PR, I will go ahead and approve and add the `AT: AUTOMERGE` label to get this merged.

**Pull request merged**
